docs/examples/performance/tgi-server.mdx

```diff
@@ -1,4 +1,94 @@
 ---
-title: Serve models with TGI
-description: "Description"
+title: Serve LLM models with TGI
+description: "Deploy a language model using TGI"
 ---
+
+[TGI](https://github.com/huggingface/text-generation-inference/tree/main) is a model server optimized for language models.
+
+<Tip>
+You can see the config for the finished model on the right. Keep reading for step-by-step instructions on how to generate it.
+</Tip>
+
+This example will cover:
+
+1. Generating the base Truss
+2. Setting sufficient model resources for inference
+3. Deploying the model
+
+### Step 1: Generating the base Truss
+
+Get started by creating a new Truss:
+
+```sh
+truss init --backend TGI opt125
```

Comment:
Should this be `truss init opt125 --backend TGI` or does it not matter?

Reply:
it does not matter. I like the name being at the end, but it doesn't matter